dynawo/3rdParty/omcUpdate_1_9_4/Makefile.in.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile.in b/Makefile.in
+index b181ae212..dd42ce615 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -122,7 +122,7 @@ install: install-dirs
+ 	# License
+ 	cp -p OMCompiler/COPYING ${INSTALL_DOCDIR}/omc/
+ 	# Documents
+-	test ! -e "doc" || cp -p doc/*.pdf ${INSTALL_DOCDIR}/omc/
++	test -e "doc" || cp -p doc/*.pdf ${INSTALL_DOCDIR}/omc/
```

Comment:
What is the gain of this? It would be good to limit the patch to the strict minimum. If it is only for performance or generation of useless files I propose to keep the original Makefile to avoid having hard time in adapting the patches to latest OM version

Reply:
The Makefile does not work otherwise